core/shared/src/test/scala/sigma/VersionTesting.scala

```diff
@@ -41,9 +39,10 @@ trait VersionTesting {
           _ + 1) { j =>
           val treeVersion = ergoTreeVers(j)
           // for each tree version up to currently activated, set it up and execute block
-          _currErgoTreeVersion.withValue(treeVersion)(block)
+          _currErgoTreeVersion.withValue(treeVersion) {
+            VersionContext.withVersions(activatedVersion, treeVersion)(block)
```

Comment:
Why this "VersionContext.withVersions" call is necessary? The activatedVersion has already been set above.